css/mobile.css

```diff
@@ -0,0 +1,157 @@
+.nav-logo {
+  padding: 0;
+}
+
+.nav-menu {
+  display: none;
+}
+
+.nav-menu.active {
+  left: 0;
+}
+
+.nav-item {
+  margin: 40px 0;
+}
```

Comment:
- Kindly use your media screens into your project.
@media screen and (min-width: 990px) {
/*
.
.
*/
}

Reply:
It's already in there. 11th line in index.html
`<link rel="stylesheet" href="css/mobile.css" media="only screen and (max-width: 768px)">`